### DISCUSSION TITLE
Welcome to Go-bjut Discussions!

### DISCUSSION THREAD

**Discussion by yahuian** [YahuiAn/Go-bjut#8] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GO-Loc-GO** [comment]
I’m actually thinking that maybe something similar can be built for BJUT students.
[cid:image003.png@01D6D17F.4B15A5D0]

发送自 Windows 10 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>应用

发件人: YahuiAn<mailto:notifications@github.com>
发送时间: 2020年12月13日 16:57
收件人: YahuiAn/Go-bjut<mailto:Go-bjut@noreply.github.com>
抄送: Subscribed<mailto:subscribed@noreply.github.com>
主题: [YahuiAn/Go-bjut] Welcome to Go-bjut Discussions! (#8)

👋 Welcome!

We’re using Discussions as a place to connect with other members of our community. We hope that you:

  *   Ask questions you’re wondering about.
  *   Share ideas.
  *   Engage with other community members.
  *   Welcome others and are open-minded. Remember that this is a community we
build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Reply by yahuian** [reply]
可以说来听听~
